kmod/patch/Makefile

```diff
@@ -1,17 +1,18 @@
 KPATCH_NAME ?= patch
 KPATCH_BUILD ?= /lib/modules/$(shell uname -r)/build
 KPATCH_MAKE = $(MAKE) -C $(KPATCH_BUILD) M=$(PWD)
+KPATCH_TYPE ?= kpatch
 
-obj-m += kpatch-$(KPATCH_NAME).o
+obj-m += $(KPATCH_TYPE)-$(KPATCH_NAME).o
 
-kpatch-$(KPATCH_NAME)-objs += patch-hook.o kpatch.lds output.o
+$(KPATCH_TYPE)-$(KPATCH_NAME)-objs += patch-hook.o $(KPATCH_TYPE).lds output.o
 
-all: kpatch-$(KPATCH_NAME).ko
+all: $(KPATCH_TYPE)-$(KPATCH_NAME).ko
 
-kpatch-$(KPATCH_NAME).ko:
-	$(KPATCH_MAKE) kpatch-$(KPATCH_NAME).ko
+$(KPATCH_TYPE)-$(KPATCH_NAME).ko:
+	$(KPATCH_MAKE) $(KPATCH_TYPE)-$(KPATCH_NAME).ko
 
-patch-hook.o: patch-hook.c kpatch-patch-hook.c livepatch-patch-hook.c
+patch-hook.o: patch-hook.c
```

Comment:
Yes, but the dependency needs to be specified so that make knows to rebuild patch-hook.o if one of the source files changes.